src/shared/firewall-util-nft.c

```diff
@@ -924,6 +930,29 @@ static int fw_nftables_add_masquerade_internal(
         return sd_nfnl_call_batch(nfnl, &m, 1, NFNL_DEFAULT_TIMEOUT_USECS, NULL);
 }
 
+static int af_to_nfproto(int af) {
+        assert(IN_SET(af, AF_INET, AF_INET6));
+
+        switch(af) {
```

Comment:
Please insert a space between `switch` and `(`.

Reply:
whoops